### PR TITLE
Add Translations to Contributing.md

### PR DESCRIPTION
**Pull request by Chefski**
---

- [x] I understand that contributing to this repository will require me to agree with the [CLA](https://github.com/anyproto/.github/blob/main/docs/CLA.md)

---

### Description
Added a subheading for translations in the contributing guidelines which redirects to the Crowdin page.

### What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [x] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI

### Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

### Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 [tech-docs](https://github.com/anyproto/tech-docs)
- [ ] 🙅 no documentation needed

### CODE REVIEW

**Comment by Chefski** [comment]
I noticed that anytpe-ts has localisation on the `README.md` page instead of `CONTRIBUTING.md`. Let me know if it would be better to move it to `README.md` and change the heading name to "Localisation"

**Comment by Chefski** [comment]
I have read the CLA Document and I hereby sign the CLA

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -109,6 +109,10 @@ Basic rules for pull requests:
 
 All pull requests will be reviewed by the team.
 
+### Translations
```
Comment (fuksman) [inline docs/CONTRIBUTING.md:112]: Let's rename this to `Localization`

Comment (fuksman) [reply docs/CONTRIBUTING.md:112]: Can you also update the table of contents?

**Review by fuksman — CHANGES_REQUESTED** [review]

File: docs/CONTRIBUTING.md (hunk 1 above)
Comment (Chefski) [reply docs/CONTRIBUTING.md]: Whoops 😅 Added it now and renamed :)

**Review by fuksman — APPROVED** [review]

**Comment by fuksman** [comment]
@any contributor @Chefski docs

**Pull request merged**